plot.py

```diff
@@ -774,12 +778,12 @@ def _pass_lims(self):
         self.LimsDialog.accept()
         self.Vlims.change(self.limits)
              
-    def NewLimits(self, variable, value):
+    def NewLimits(self, variable, value, strong):
         '''Retrieve new limits input'''
         #self._pass_lims()
         print "In NewLims"
     
-    def NewRadar(self, variable, value):
+    def NewRadar(self, variable, value, strong):
```

Comment:
It looks like NewRadar and NewLimits are outdated after ZoomPan update.  Just making a note so that I hope to remember to delete in a future push.